py/torch_tensorrt/dynamo/conversion/impl/grid.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional, Sequence
+
+import tensorrt as trt
+import torch
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
+from torch_tensorrt.dynamo.conversion.converter_utils import (
+    GridSamplerInterpolation,
+    GridSamplerSampling,
+    cast_trt_tensor,
+)
+from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+
+
+def grid(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    grid: TRTTensor,
+    interpolation_mode: int,
+    padding_mode: int,
+    align_corners: bool,
+    output_mask: Optional[Sequence[bool]] = None,
+) -> TRTTensor:
+    grid_layer = ctx.net.add_grid_sample(input, grid)
+    interpolation_mode_trt = GridSamplerInterpolation()
+    grid_layer.interpolation_mode = interpolation_mode_trt(interpolation_mode)
+    sample_mode_trt = GridSamplerSampling()
```

Comment:
Consider using a dictionary for this - could be `grid_layer.interpolation_mode = Dictionary.get(interpolation_mode, None)`